misc/validators/validator-puppet

```diff
@@ -18,6 +18,32 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,
 # USA.
 #
+#
+# NOTE about puppet version
+# ------------------------------------------------------------------------
+# This driver script doesn't take care the version of puppet.
+# As the result, you may get much warnings on some platform
+# though the results of validations are "valid".
+#
```

Comment:
This driver script does not specify the used puppet version. Instead, puppet version will depend on the platform runtime.  As a result, you may see some warnings on some platforms even tough the validation has passed.